tools/list-dependencies.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+
+#
+# Copyright contributors to the Galasa project
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+#-----------------------------------------------------------------------------------------                   
+#
+# Objectives: Build this repository code locally.
```

Comment:
Do you want to update this for this script to avoid confusion

Reply:
fixed